build/config/environments.js

```diff
@@ -3,10 +3,10 @@ export default {
 
     production: (config) => ({
         devtool: false,
-        publicPath: '/',
-        assetsNameJs: 'js/[name].[hash].js',
-        assetsNameImg: 'img/[name].[chunkhash].[ext]',
-        assetsNameCss: 'css/[name].[chunkhash].css',
-        assetsNameFont: 'fonts/[name].[hash].[ext]',
+        public_path: '/',
```

Comment:
Check if in prod we have `<script src="{{ head.resourcesUrl }}/js/manifest.aeed17450313dc30ea33.js" type="text/javascript"></script>` tag alike in production.